config/sidebar.paper.ts

```diff
@@ -143,7 +152,6 @@ const paper: SidebarsConfig = {
             },
             "dev/api/pdc",
             "dev/api/custom-inventory-holder",
-            "dev/api/commands",
```

Comment:
needs a redirect

Reply:
why? we have the new sidebar item, right?

Reply:
it breaks existing URLs